js/packages/teams-ai/src/models/OpenAIModel.ts

```diff
@@ -400,8 +402,19 @@ export class OpenAIModel implements PromptCompletionModel {
                     console.log(Colorize.value('duration', Date.now() - startTime, 'ms'));
                 }
             } else {
-                const actionCalls: ActionCall[] = [];
                 const responseMessage = (completion as ChatCompletion).choices![0].message;
+                message = {
+                    role: responseMessage.role,
+                    content: responseMessage.content ?? ''
+                };
+
+                // Preserve message context if there is any
+                const messageWithContext = responseMessage as Message<string>;
```

Comment:
Is it necessary to typecast this? Is `context` not part of the original `responseMessage` type.

Reply:
Yes, the typecast is necessary because the original OpenAI type doesn't include context. 
I could do `const messageWithContext: Message<string> = responseMessage as Message<string>` if that feels better?

Reply:
the way you have it is fine as well. thanks